<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Labrad Connect</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/DEMONSResources.qrc">
    <normaloff>:/Pictures/Pictures/DemonLogo.png</normaloff>:/Pictures/Pictures/DemonLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
background-color: rgb(0, 0, 0);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ConnectAll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_ConnectAll{
color: rgb(168,168,168);
background: rgb(60, 60, 60);
border-radius: 4px;
}

QPushButton:pressed#pushButton_ConnectAll{
color: rgb(168,168,168);
background-color:rgb(100,100,100);
border-radius: 4px
}</string>
         </property>
         <property name="text">
          <string>Connect All Servers</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DisconnectAll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_DisconnectAll{
color: rgb(168,168,168);
background: rgb(60, 60, 60);
border-radius: 4px;
}

QPushButton:pressed#pushButton_DisconnectAll{
color: rgb(168,168,168);
background-color:rgb(100,100,100);
border-radius: 4px
}</string>
         </property>
         <property name="text">
          <string>Disconnect All Servers</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>381</width>
        <height>121</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame{
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="11" column="10">
        <widget class="QLabel" name="label_AMI430">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>GPIB Server:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="10">
        <widget class="QLabel" name="label_SerialServer">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>LabRAD:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="8">
        <widget class="QPushButton" name="pushButton_Keithley2450">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_Keithley2450{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="8">
        <widget class="QPushButton" name="pushButton_AMI430">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_AMI430{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="4">
        <widget class="QLabel" name="label_Lakeshore">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="3">
        <widget class="QPushButton" name="pushButton_Lakeshore">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_Lakeshore{
background:rgb(144, 140, 9);
border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QLabel" name="label_DataVault">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <widget class="QLabel" name="label_DACADC">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="10">
        <widget class="QLabel" name="label_Keithley2450">
         <property name="styleSheet">
          <string notr="true">color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>GPIB Device Manager:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="7">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Keithley 2450&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="pushButton_DataVault">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_DataVault{
background:rgb(144, 140, 9);
border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="6">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="6">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="7">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>SR830</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButton_LabRAD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_LabRAD{
background:rgb(144, 140, 9);
border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="10">
        <widget class="QLabel" name="label_GPIBServer">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Serial Server:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="8">
        <widget class="QPushButton" name="pushButton_SR830">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_SR830{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="6">
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="6">
        <spacer name="horizontalSpacer_17">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="8">
        <widget class="QPushButton" name="pushButton_SerialServer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_SerialServer{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QPushButton" name="pushButton_DACADC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_DACADC{
background:rgb(144, 140, 9);
border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="8">
        <widget class="QPushButton" name="pushButton_GPIBServer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_GPIBServer{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Lakeshore</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="label_GPIBDeviceManager">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_Labrad">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="10">
        <widget class="QLabel" name="label_SR830">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>DAC ADC:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="7">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>AMI430</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Data Vault:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="pushButton_GPIBDeviceManager">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_GPIBDeviceManager{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="3">
        <widget class="QPushButton" name="pushButton_Levelmeter">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(144, 140, 9);
border-radius: 4px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="4">
        <widget class="QLabel" name="label_Levelmeter">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">
color:rgb(168,168,168)</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Levelmeter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(168, 168, 168);
font: 10pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-wordWrap: true;
</string>
        </property>
        <property name="text">
         <string>Datavault Folder:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_DataVaultFolder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_DataVaultFolder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_DataVaultFolder{
image:url(:/Pictures/Pictures/SessionSelect.png);
background: black;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(168, 168, 168);
font: 10pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: 'AlignVCenter | AlignRight';
qproperty-wordWrap: true;
</string>
        </property>
        <property name="text">
         <string>Session Folder:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_SessionFolder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_SessionFolder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_SessionFolder{
image:url(:/Pictures/Pictures/SessionSelect.png);
background: black;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/DEMONSResources.qrc"/>
 </resources>
 <connections/>
</ui>
